Basic OOP/BasicOOP/Interface/IBaseInterface.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
```

Comment:
Let's have the only Driver in this assembly. Please move nondriver related code to the separate project